### PR TITLE
Issue1245 thermal properties materials

### PR DESCRIPTION
**Pull request by jelgerjansen**
Update of material properties in `IDEAS.Buildings.Data.Materials` and added references. This closes issue #1245.

### CODE REVIEW

File: IDEAS/Buildings/Data/Insulation/Glasswool.mo [hunk 1]
```diff
@@ -1,16 +1,12 @@
 within IDEAS.Buildings.Data.Insulation;
 record Glasswool = IDEAS.Buildings.Data.Interfaces.Insulation (
-    k=0.040,
-    c=840,
-    rho=80) "Glasswool" annotation (Documentation(info="<html>
-<p>
-Glass wool insulation thermal properties.
-</p>
+    k=0.035,
```
Comment (Mathadon) [inline IDEAS/Buildings/Data/Insulation/Glasswool.mo:3]: Interesting, I thought that 0.04 was more accurate but commercial data sheets seem to confirm this value.

File: IDEAS/Buildings/Data/Materials/GypsumBoard.mo [hunk 2]
```diff
@@ -0,0 +1,17 @@
+within IDEAS.Buildings.Data.Materials;
+record GypsumBoard = IDEAS.Buildings.Data.Interfaces.Material (
+    k=0.16,
```
Comment (Mathadon) [inline IDEAS/Buildings/Data/Materials/GypsumBoard.mo:3]: image

close enough! and a lot closer than the old value of 0.6..

File: IDEAS/Buildings/Data/Materials/Screed.mo [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
-    k=0.6,
+    k=0.45,
     c=840,
     rho=1100,
     epsLw=0.88,
-    epsSw=0.55) "Light cast concrete" annotation (Documentation(info="<html>
+    epsSw=0.55) "Typical (Belgian) screed" annotation (Documentation(info="<html>
 <p>
-Thermal properties of screed.
+Thermal properties of screed. The values are based on those provided in <a href=\"https://www.rsprojects.eu/images/Duremit/Warmtegeleiding%20van%20Duremitchape.pdf\">this document</a>.
```
Comment (Mathadon) [inline IDEAS/Buildings/Data/Materials/Screed.mo:9]: Heb je nog bronnen vergeleken?

**Comment by Mathadon** [comment]
@jelgerjansen great update! Good that you removed some older records and added new ones. 

 I have fixed the documentation in [de654b1](https://github.com/open-ideas/IDEAS/pull/1268/commits/de654b1ad35b9827ab73a14d7bce7a91c13704a1).

Final comment: did you compare multiple sources for the properties of screed? The documentation only lists one commercial source.

**Comment by Mathadon** [comment]
Running unit tests.

**Comment by Mathadon** [comment]
> Remarkable results:
> 
> image

The conductivity of Argon changed by 7%, which explains the result changes. Furthermore the result is now closer to the actual norm U value of 1.028 W/K.

File: IDEAS/Buildings/Data/Materials/Screed.mo (hunk 3 above)
Comment (jelgerjansen) [reply IDEAS/Buildings/Data/Materials/Screed.mo:9]: @Mathadon I did a new search on the web, where I focused on 'cement screed' (other types exist like liquid screed, anhydrite screed, insulating screed, ... each with its own specific thermal properties). I found the following data:
- rho=1900, k=1.2
https://www.rsprojects.eu/images/Duremit/Warmtegeleiding%20van%20Duremitchape.pdf
- k=1.3
https://www.joostdevree.nl/shtmls/warmtegeleidingscoefficient.shtml
- rho=2000, k=1.4
https://buildex.techinfus.com/en/uteplenie/teploprovodnost-uteplitelej.html
- rho=2100, k=1.95, c=900
Rauch, Michael & Werther, Norman & Winter, Stefan. (2020). FIRE DESIGN METHODS FOR TIMBER FLOOR ELEMENTS THE CONTRIBUTION OF SCREED FLOOR TOPPINGS TO THE FIRE RESISTANCE. 10.13140/RG.2.2.16032.00004. 
- rho=1800, k=1.35, c=1000
Gagliano, Antonio & Nocera, Francesco & Detommaso, Maurizio & Agrifoglio, Antonio & Patania, Francesco. (2016). Dynamic simulations and experimental measurements on an extensive green roof in Mediterranean climate. 
- rho=1200, k=1, c=1250
Values used for the buildings of De Schipjes

Based on these resources, I would opt for the following properties:
**rho=2000, k=1.3, c=1000**

Comment (Mathadon) [reply IDEAS/Buildings/Data/Materials/Screed.mo:9]: Ok! Can you summarise the search in the documentation? Or add a permalink to your post in the documentation:
image

Comment (jelgerjansen) [reply IDEAS/Buildings/Data/Materials/Screed.mo]: Done! Remark that the new values are close to those of concrete (rho=2240, k=1.4, c=900).

Comment (Mathadon) [reply IDEAS/Buildings/Data/Materials/Screed.mo]: Strange.. but ok!

**Comment by Mathadon** [comment]
@jelgerjansen this has been merged! thanks for doing most of the work for this one!

**Pull request merged**